CMakeLists.txt

```diff
@@ -114,7 +114,7 @@ set(PYTHON_CONFIGURE_OPTIONS --enable-ipv6 --with-threads --with-fpectl)
 
 ## Linux specific options
 if(LINUX)
-    set(PYTHON_OPENSSL "CPPFLAGS=-I${OPENSSL_DIR}" "LDFLAGS=-L${OPENSSL_DIR}/lib -L/usr/lib/x86_64-linux-gnu")
+    set(PYTHON_OPENSSL "CPPFLAGS=-I${OPENSSL_DIR}" "LDFLAGS=-L${OPENSSL_DIR}/lib -L/usr/lib64")
```

Comment:
I think we might need some logic here for different distributions, Ubuntu doesn't have /usr/lib64.